src/panel.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8">
+    <title>Block-facebook</title>
+    <link rel="stylesheet" href="panel.css">
+  </head>
+  <body>
+    <div id="panel-1" class="wrapper">
+      <h4 class="profile heading padding">
+        Facebook Container
+      </h4>
+      <p class="padding">
+        Facebook Container has been installed and is preventing Facebook from tracking you around the web. Anytime you see <img src="fb-ban2.png"> this add-on is blocking Facebook from tracking you.
```

Comment:
Also, are these the correct icons? I tried verifying against the Invision screens in #203, but I think I counted at least 5 different toolbar icons for Facebook Container in that deck, so I wasn't sure what the final ones were.

Reply:
These are test icons only. The final one needs a platform level change.